Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalTrg_Output_cff.py

```diff
@@ -0,0 +1,96 @@
+import FWCore.ParameterSet.Config as cms
+
+OutALCARECOEcalTrg_noDrop = cms.PSet(
+    # put this if you have a filter
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOEcalTrg'),
+    ),
+    outputCommands = cms.untracked.vstring( 
+        'CaloTowersSorted_towerMaker_*',
```

Comment:
yes.... just realized. I've fixed that and running the runTheMatrix to make sure now there are no other problems....

-- 
 							Shervin


On Mon, 30 May 2016, Marco Musich wrote:

> Date: Mon, 30 May 2016 18:49:48 +0200
> From: Marco Musich <notifications@github.com>
> Reply-To: cms-sw/cmssw
>     <reply+0045c225ea74ca34bd4e55dd767cf9506c657639cb178a8f92cf000000011364312
>     c92a170ce03e13aca@reply.github.com>
> To: cms-sw/cmssw <cmssw@noreply.github.com>
> Cc: shervin86 <shervin@cern.ch>, Mention <mention@noreply.github.com>
> Subject: Re: [cms-sw/cmssw] Alcareco ecal trg (#14684)
> 
>> @@ -0,0 +1,96 @@
>> +import FWCore.ParameterSet.Config as cms
>> +
>> +OutALCARECOEcalTrg_noDrop = cms.PSet(
>> +    # put this if you have a filter
>> +    SelectEvents = cms.untracked.PSet(
>> +        SelectEvents = cms.vstring('pathALCARECOEcalTrg'),
>> +    ),
>> +    outputCommands = cms.untracked.vstring(
>> +        'CaloTowersSorted_towerMaker_*',
>
> @shervin86  you need to include the `keep` and `drop` statements.
> ```
> ----- Begin Fatal Exception 30-May-2016 14:57:08 CEST-----------------------
> An exception of category 'Configuration' occurred while
>   [0] Constructing the EventProcessor
>   [1] Constructing module: class=PoolOutputModule label='ALCARECOStreamEcalTrg'
> Exception Message:
> Invalid statement in configuration file
> In OutputModule parameter named 'outputCommands'
> Rule must specify 'keep ' or 'drop ' and also supply a pattern.
> This is the invalid output configuration rule:
>    CaloTowersSorted_towerMaker_*
> Exception thrown from ProductSelectorRules::Rule
> ----- End Fatal Exception -------------------------------------------------
> ```
>
>
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-sw/cmssw/pull/14684/files/05999d6d54d5abf13199e6f5d8e362b14111654b#r65092298
>